terragrunt/aws/vpc/vpc.tf

```
@@ -0,0 +1,38 @@
+module "vpc" {
+  source            = "github.com/cds-snc/terraform-modules?ref=v0.0.46//vpc"
+  name              = "efs-vpc"
+  billing_tag_value = "CostCentre"
+  high_availability = true
+  enable_flow_log   = true
+  block_ssh         = true
+  block_rdp         = true
+
+  allow_https_request_out          = true
+  allow_https_request_out_response = true
+  allow_https_request_in           = true
+  allow_https_request_in_response  = true
+
```

Could you also set this attribute to save a bit of money (NAT gateways are charged per hour running, and this app can make do with only 1):
```suggestion
  single_nat_gateway = true
```